arrow-libs/suspendapp/suspendapp/src/commonMain/kotlin/arrow/continuations/SuspendApp.kt

```diff
@@ -17,38 +17,55 @@ import kotlinx.coroutines.*
  * @param timeout the maximum backpressure time that can be applied to the process. This emulates a
  *   `SIGKILL` command, and after the [timeout] is passed the App will forcefully shut down
  *   regardless of finalizers.
+ * @param forceExit whether to force-exit the process after this SuspendApp
+ *   finishes (code 0 on success or 255 on failure)
  * @param block the lambda of the actual application.
  */
-@OptIn(ExperimentalStdlibApi::class)
 public fun SuspendApp(
   context: CoroutineContext = Dispatchers.Default,
   uncaught: (Throwable) -> Unit = Throwable::printStackTrace,
   timeout: Duration = Duration.INFINITE,
   process: Process = process(),
+  forceExit: Boolean = true,
```

Comment:
this is an escape-hatch for if you want to opt-out of the `env.exit` calls

we may or may not want to keep this in the PR

Reply:
actually I'll remove this for now as it's a breaking change on the API